ulib/ulibfs.fsproj

```diff
@@ -63,10 +63,16 @@
     <Compile Include="fs\FStar_ST.fs" />
     <Compile Include="fs\FStar_All.fs" />
     <Compile Include="fs\FStar_IO.fs" />
+    <Compile Include="fs\FStar_Pervasives_Native.fs" />
 
     <FStarSourceFile Include="FStar.Pervasives.fst" />
     <FStarInterfaceFile Include="FStar.Pervasives.fsti" />
 
+    <FStarSourceFile Include="FStar.List.Tot.Base.fst" />
+
+    <FStarSourceFile Include="FStar.Seq.Base.fst" />
+    <FStarInterfaceFile Include="FStar.Seq.Base.fsti" />
```

Comment:
@nikswamy It looks like preserving the order of "FStarSourceFile" items when generating fs files instead of getting the compilation order from a dependency analysis run could also work. At least for now.